charts/prometheus-blackbox-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 7.2.0
+version: 7.2.1
```

Comment:
Is this still a patch using appVersion as default?
Or do you want to see a minor/major update now?

Reply:
A patch is okay since it is not breaking and is not a new feature!